dump/send_pldm_cmd.cpp

```diff
@@ -5,6 +5,8 @@
 #include <fmt/format.h>
 #include <libpldm/file_io.h>
 
+#include <format>
+#include <iostream>
```

Comment:
we can remove iostream inclusion. And if sstream inclusion is needed only for this, we can remove that one too.

Reply:
+1